Documentation/ColumnsConfig/Type/Group/Index.rst

```diff
@@ -6,23 +6,23 @@
 Group fields
 ============
 
-The group element in TYPO3 makes it possible to create references to records from multiple tables in the system.
+The group element (:php:`type' == 'group'`) in TYPO3 makes it possible to create references from a record of one table to many records from multiple tables in the system.
```

Comment:
```suggestion
The group element (:php:`type' => 'group'`) in TYPO3 makes it possible to create references from a record of one table to many records from multiple other tables in the system.
```

Reply:
It can also be a related between to tt_content tables. So both tables can be the same.

Reply:
True, I've adjusted the suggestion to make this clearer.

Reply:
instead of "The foreign table(s)"
better:
The foreign table

This would be understandable.

Reply:
Well, there can be many foreign tables ...

Reply:
But please remove the (s). This does not look good.

Reply:
Sure, done now. :+1: